scripts/index.js

```diff
@@ -1,8 +1,6 @@
 const movieResultsElement = document.getElementById("movie-results")
 const searchInput = document.getElementById("search-input")
 const searchBtn = document.getElementById("search-btn")
-let movieResults = []
```

Comment:
As we create `movieResuts` inside the `searchMovieDatabase` function, and that function eventually calls all the other functions that rely on `movieResults`, we can remove this global variable and just pass the `movieResults` down into other functions as arguments and parameters